library/entity/entity_events/src/main/java/org/quiltmc/qsl/entity_events/api/EntityReviveEvents.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright 2021-2022 QuiltMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.quiltmc.qsl.entity_events.api;
+
+import net.minecraft.entity.LivingEntity;
+import net.minecraft.entity.damage.DamageSource;
+import org.quiltmc.qsl.base.api.event.Event;
+import org.quiltmc.qsl.base.api.event.EventAwareListener;
+
+/**
+ * A pair of callbacks which are invoked when a LivingEntity takes fatal damage.
+ *
+ * <p>Mods can return true from a callback to keep the entity alive, like totems of undying do. This will also prevent
```

Comment:
```suggestion
 * <p>
 * Mods can return true from a callback to keep the entity alive, like totems of undying do. This will also prevent
```
Actually, i'm not gonna do a heckton of reviews like that here; just please use checkstyle and fix all the errors

Reply:
Checkstyle has been run and everything it flagged up fixed. Interestingly it didn't flag up these javadoc issues.